proc_macros/src/rule_tests.rs

```diff
@@ -319,12 +329,13 @@ impl ToTokens for InvalidRuleTestSpec {
         let code = &self.code;
         let errors = self.errors.iter().map(|error| quote!(#error));
         let output = match self.output.as_ref() {
+            Some(output) if expr_is_ident(output, "None") => quote! {
+                Some(tree_sitter_lint::RuleTestExpectedOutput::NoOutput)
```

Comment:
The context here is that we are mimicking ESLint where if a rule test specifies `output: null` it is asserting that no fixing took place

So this is supporting what I'd think is the natural syntax for specifying that which is `output => None`

But it's doing some special "massaging" of that case because in reality we are storing that as `Some(RuleTextExpectedOutput::NoOutput)` (where `None` means "nothing related to checking the output was specified)